7.4/Dockerfile-alpine

```diff
@@ -0,0 +1,253 @@
+
+FROM build-base AS build-intl
+RUN apk add --no-cache icu-dev
+RUN docker-php-ext-install intl
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-mysqli
+RUN docker-php-ext-install mysqli
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-pdo_mysql
+RUN docker-php-ext-install pdo_mysql
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-opcache
+RUN docker-php-ext-install opcache
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-bcmath
+RUN docker-php-ext-install bcmath
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-calendar
+RUN docker-php-ext-install calendar
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-sockets
+RUN docker-php-ext-install sockets
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-redis
+RUN pecl install redis
+RUN docker-php-ext-enable redis
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-ldap
+RUN apk add --no-cache openldap-dev
+RUN docker-php-ext-install ldap
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-pcntl
+RUN docker-php-ext-configure pcntl --enable-pcntl
+RUN docker-php-ext-install pcntl
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-soap
+RUN apk add --no-cache libxml2-dev
+RUN docker-php-ext-install soap
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-memcached
+RUN apk add --no-cache  zlib-dev libmemcached-dev
+RUN pecl install memcached
+RUN docker-php-ext-enable memcached
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-imagick
+RUN apk add --no-cache imagemagick-dev
+RUN pecl install imagick
+RUN docker-php-ext-enable imagick
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-pgsql
+RUN apk add --no-cache postgresql-dev
+RUN docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql
+RUN docker-php-ext-install pgsql
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-pdo_pgsql
+RUN apk add --no-cache postgresql-dev
+RUN docker-php-ext-install pdo_pgsql
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-pcov
+RUN pecl install pcov
+RUN docker-php-ext-enable pcov
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM build-base AS build-xdebug
+RUN pecl install xdebug
+RUN docker-php-ext-enable xdebug
+RUN cp `php -i | grep ^extension_dir | cut -f 3 -d ' '`/*.so /tmp/
+
+FROM base
+
+ENV COMPOSER_HOME=/composer
+COPY --from=composer:1.8.6 /usr/bin/composer /usr/local/bin/composer
```

Comment:
1.9.1

Reply:
please fix it upstream. it's still 1.8.6 in the non-alpine images, too.